Dockerfile

```diff
@@ -1,26 +1,31 @@
-FROM centos:7
+FROM selenium/standalone-chrome:2.46.0
 
-RUN curl https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein > /usr/bin/lein
-RUN chmod 755 /usr/bin/lein
+MAINTAINER Silver Team <silver@broadinstitute.org>
 
-RUN echo -e '[google-chrome]\n\
-name=google-chrome\n\
-baseurl=http://dl.google.com/linux/chrome/rpm/stable/$basearch\n\
-enabled=1\n\
-gpgcheck=1\n\
-gpgkey=https://dl-ssl.google.com/linux/linux_signing_key.pub\n\
-' > /etc/yum.repos.d/google-chrome.repo
+USER root
 
-RUN yum -y install  google-chrome-stable unzip GConf2 java-1.8.0-openjdk && yum clean all
+# Basics
+RUN apt-get update && apt-get install -y \
+ git \
+ curl \
+ xvfb \
+ wget
```

Comment:
its helpful to clean this up after (otherwise the images end up pretty big):

```    
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* 
```